### PR TITLE
ROC-2863 CNP infra pipeline

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  source = "git@github.com/contino/moj-module-webapp.git?ref=0.0.78"
+  product = "${var.product}-claim-store"
+  location = "${var.location}"
+  env = "${var.env}"
+  asename = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+
+  app_settings = {
+    CLAIM_STORE_DB_HOST = "${module.claim-store-database.host_name}"
+    CLAIM_STORE_DB_PORT = "${module.claim-store-database.postgresql_listen_port}"
+    POSTGRES_DATABASE = "${module.claim-store-database.postgresql_database}"
```
Comment (timja) [inline main.tf]: will need to change something to make this work

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+module "claim-store-database" {
+  source = "git@github.com/contino/moj-module-postgres?ref=master"
+  product = "${var.product}"
+  location = "West Europe"
+  env = "${var.env}"
+  postgresql_user = "claimstore"
+  postgresql_password = "Password1" // TODO: keyvault?
```
Comment (timja) [inline main.tf:50]: how to keyvault this? rhubard hardcodes the password

File: state.tf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+terraform {
+  backend "azurerm" {}
+}
+
+data "terraform_remote_state" "core_apps_infrastructure" {
+  backend = "azurerm"
+
+  config {
```
Comment (timja) [inline state.tf:8]: is this config right?

File: terraform.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+state_store_resource_group="contino-moj-tf-state"
```
Comment (timja) [inline terraform.sh:3]: I assume this file is for local testing?

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,33 @@
+variable "product" {
+  type    = "string"
+  default = "cmc"
+}
+
+variable "qaslotname" {
```
Comment (timja) [inline variables.tf]: what is a slot?

File: main.tf [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  product = "${var.product}-claim-store"
+  location = "${var.location}"
+  env = "${var.env}"
+  asename = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+
+  app_settings = {
+    CLAIM_STORE_DB_HOST = "${module.claim-store-database.host_name}"
+    CLAIM_STORE_DB_PORT = "${module.claim-store-database.postgresql_listen_port}"
+    POSTGRES_DATABASE = "${module.claim-store-database.postgresql_database}"
+    CLAIM_STORE_DB_USERNAME = "${module.claim-store-database.user_name}"
```
Comment (rkondratowicz) [inline main.tf]: apparently
>`user_name` -  the username given in postgresql_user combined with the server name in the format postgresql_user@postgres-paas.name

Is that what you expect?

**Pull request merged**